src/overlays/actors/ovl_En_Bb/z_en_bb.c

```diff
@@ -1083,14 +1083,14 @@ void EnBb_SetupStunned(EnBb* this) {
     }
     switch (this->dmgEffect) {
         case 8:
-            Actor_SetColorFilter(&this->actor, -0x8000, 0xC8, 0, 0x50);
+            Actor_SetColorFilter(&this->actor, -COLORFILTER_COLORFLAG_GRAY, 200, COLORFILTER_BUFFLAG_OPA, 80);
```

Comment:
about `-COLORFILTER_COLORFLAG_GRAY`, I'm on board with just using `COLORFILTER_COLORFLAG_GRAY` instead :shrug:

Reply:
I swear I tried this before and it didn't match, but now it does, I guess I did something wrong before